.github/actions/validate-apiml-healthy/action.yml

```diff
@@ -29,8 +37,8 @@ runs:
             valid="false"
             until [ $attemptCounter -gt $maxAttempts ]; do
               sleep 10
-              gwHealth=$(curl -k -s https://localhost:10010/application/health)
-              zaasHealth=$(curl -k -s https://localhost:10023/application/health)
+              gwHealth=$(curl -k -s https://${{ inputs.gateway-host }}:10010/application/health)
```

Comment:
why is this required?

Reply:
it's not, I was playing around with this action to use it but at least now it's parameterizable